pipelines/h6c/rtp/v2/stage_1_task_scripts/do_ALL_SKY_IMAGE_MOVIE.sh

```diff
@@ -0,0 +1,30 @@
+set -e
+
+# This script generates an HTML version of a notebook summarizing the output of auto_metrics, ant_metrics, and redcal chisq
+
+src_dir="$(dirname "$0")"
+source ${src_dir}/_common.sh
+
+# Parameters are set in the configuration file. Here we define their positions,
+# which must be consistent with the config.
+# 1 - (raw) filename
+# 2 - nb_template_dir: where to look for the notebook template
+# 3 - nb_output_repo: repository for saving evaluated notebooks
+# 4 - git_push: boolean whether to push the results created in the nb_output_repo
+# 5 - good_statuses: string list of comma-separated (no spaces) antenna statuses considered "good"
+
+fn=${1}
+
+# Get JD from filename
+jd=$(get_int_jd ${fn})
+
+
+
+#image stack is in its own dir because glob dereferencing is hard
+cd ${jd}_imgs
+ffmpeg -f image2 -pattern_type glob -i '*.jpg' HERA_${jd}_automatic.mp4
+cp  HERA_${jd}_automatic.mp4 ..
+# TODO put movie in Librarian and upload to enterprise for quick online viewing
+cd ..
```

Comment:
As it stands, these movies and images don't go anywhere. We need a librarian step or a github upload or both.

Reply:
correct. Was aiming to get a test run done first but why not, we might as well set up the whole thing at once. 
I was thinking of a cron job rsync to enterprise and a backup step to librarian. How does that sound?

Reply:
for now movies finished movie to /sn1/nightly_movies 
a cronjob will attempt to rsync this directory twice a day to heranow.
TODO: add movies to the librarian.